### PR TITLE
SUSI Line Bot Added

### PR DESCRIPTION
**Pull request by AliAyub007**
Added SUSI bot for Line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,82 @@
...
+- Select either Start using Messaging API.
+- Enter the required information for your new LINE@ account. Confirm the information.
+- Click the LINE@ MANAGER button at the bottom of the confirmation page.
+- Go to the settings and then  Bot Settings page of the LINE@ Manager then click Enable API.
+
+![LINE_LOGIN](/Assets/Line_Business_Center_Login.PNG)
+
+## 3- Configure your bot:
+
+- In the LINE@ Manager, go to the [Bot Settings](https://admin-official.line.me/8279988/bot-api/setting) page, which can be found under &quot;Settings&quot; on the side menu.
```
Comment (betterclever) [inline README.md]: Is this number generic or specific to your project? Like project id?
If specific to your project, better use a placeholder.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "SUSI-Bot",
+  "version": "0.0.0",
```
Comment (betterclever) [inline package.json]: Provide a version.

**Comment by mariobehling** [comment]
> Looks good, but images in Assets should not be stored in the repo. Use an external service for that.

Not sure, where this idea comes from. We are not following this policy at FOSSASIA. Relevant images should be stored in `/docs/images/`. Thanks.

**Comment by saurabhjn76** [comment]
> Not sure, where this idea comes from. We are not following this policy at FOSSASIA. Relevant images should be stored in /docs/images/. Thanks.

Sorry @mariobehling, thought this would reduce the size of a repository, will pay attention to policies followed at Fossasia.

**Comment by AliAyub007** [comment]
@mariobehling Is it fine in Assests or I should change it to docs/images ?

**Comment by chashmeetsingh** [comment]
@AliAyub007 add the screenshots under `/docs/images`

**Pull request merged**